### PR TITLE
QICore6 Uplift - CMS 249

### PR DESCRIPTION
**Pull request by soy-park**
**Uplift for CMS-249 (_Appropriate Use of DXA Scans in Women Under 65 Years Who Do Not Meet the Risk Factor Profile for Osteoporotic Fracture_) to QICore v6.0.0.**

- Updated CQL
- Updated test cases
- Attempted to execute CQL in VSCode plugin
   - Suspect there is a bug with the "Denominator" in CQL file, since it results in a "false" for every test case 
   - Error also suspects an error in FHIRHelpers.cql library 

Next steps: Fix bugs, determine expected output for test cases, and compare test case results to expected output

### CODE REVIEW

File: input/cql/AppropriateDXAScansForWomenUnder65FHIR.cql [hunk 1]
```diff
@@ -121,24 +122,27 @@ define "First BMI in Measurement Period Less Than or Equal to 20 kg m2":
...
         and AverageDrinks.value as Quantity > 2 '{drinks}/d'
       sort by start of effective.toInterval()
   )
 
+/*
+@QICore6Uplift: Per Authoring Patterns, "prevalence period would only be expected to be present on problem list items and health concerns”
+*/
 define "Has Risk Factor Any Time in History Prior to Measurement Period":
-  exists ( ( [Condition: "Osteoporosis Without Current Fracture"] OsteoporosisDiagnosis
+  exists ( ( [ConditionProblemsHealthConcerns: "Osteoporosis Without Current Fracture"] OsteoporosisDiagnosis
```
Comment (jbyrdevans) [inline input/cql/AppropriateDXAScansForWomenUnder65FHIR.cql:142]: Should be the union of the Conditions

File: input/cql/AppropriateDXAScansForWomenUnder65FHIR.cql [hunk 2]
```diff
@@ -121,24 +122,27 @@ define "First BMI in Measurement Period Less Than or Equal to 20 kg m2":
...
         where OsteoporosisDiagnosis.prevalenceInterval ( ) starts before start of "Measurement Period"
     )
-      union ( [Condition: "Osteopenia"] OsteopeniaDiagnosis
+      union ( [ConditionProblemsHealthConcerns: "Osteopenia"] OsteopeniaDiagnosis
```
Comment (jbyrdevans) [inline input/cql/AppropriateDXAScansForWomenUnder65FHIR.cql:145]: Should be the union of the Conditions

File: input/cql/FHIRHelpers.cql
Comment (jbyrdevans) [inline input/cql/FHIRHelpers.cql:1]: Are the changes to FHIRHelpers necessary?

**Review by jbyrdevans — CHANGES_REQUESTED** [review]

File: input/cql/AppropriateDXAScansForWomenUnder65FHIR.cql (hunk 1 above)
Comment (soy-park) [reply input/cql/AppropriateDXAScansForWomenUnder65FHIR.cql:142]: Based on the Authoring Patterns page (https://github.com/cqframework/CQL-Formatting-and-Usage-Wiki/wiki/Authoring-Patterns-QICore-v6.0.0), it seems that the presence of prevalenceInterval is associated with ConditionProblemsHealthConcerns.

"Problem list items and health concerns are typically documented with additional information about the condition such as prevalence period and clinical status... In particular, clinical status and prevalence period would only be expected to be present on problem list items and health concerns." Am I interpreting this correctly?

File: input/cql/AppropriateDXAScansForWomenUnder65FHIR.cql (hunk 2 above)
Comment (soy-park) [reply input/cql/AppropriateDXAScansForWomenUnder65FHIR.cql:145]: Based on the Authoring Patterns page (https://github.com/cqframework/CQL-Formatting-and-Usage-Wiki/wiki/Authoring-Patterns-QICore-v6.0.0), it seems that the presence of prevalenceInterval is associated with ConditionProblemsHealthConcerns.

"Problem list items and health concerns are typically documented with additional information about the condition such as prevalence period and clinical status... In particular, clinical status and prevalence period would only be expected to be present on problem list items and health concerns." Am I interpreting this correctly?

File: input/cql/FHIRHelpers.cql
Comment (soy-park) [reply input/cql/FHIRHelpers.cql:1]: It was showing lots of errors, so I thought I would refactor/adjust it. But, I don't mind leaving them as is until we get to a point of debugging everything?

**Review by jbyrdevans — APPROVED** [review]

File: input/cql/FHIRHelpers.cql
Comment (soy-park) [reply input/cql/FHIRHelpers.cql:1]: I didn't reference a different version of FHIRHelpers. Should I revert the changes to this?

**Pull request merged**